Completion/Unix/Command/_code

```diff
@@ -0,0 +1,36 @@
+#compdef code
+
+local arguments
+
+arguments=(
+  '(-d --diff)'{-d,--diff}'[compare two files with each other]:file to compare:_files:file to compare with:_files'
+  '(-a --add)'\*{-a,--add}'[add specified directory (or directories) to the last active window.]:directory:_directories'
+  '(-g --goto)'{-g,--goto}'[open a file at the path on the specified line and character position]:file\:line[\:character]:_files -r \:'
```

Comment:
Unlike earlier, I don't have VS code available but I wouldn't be surprised if the -g option can also be repeated. Could be worth checking.

Reply:
Nope. When I test, it seems to ignore the second `-g`